### ISSUE TITLE
"enable for this file" command

### ISSUE THREAD

**Issue by wizzard0** [ConsenSys/vscode-ethover#7]
while it's nice it's available for js, ts etc, it would also be useful when viewing log files and so on. one-time command seems like a better tradeoff than trying to guess the file formats

**Comment by tintinweb** [comment]
Yeah, the initial idea was to only have vscode load the extension when needed instead of having it idling around consuming resources. In order to change this behavior, I'd have to keep the extension awake in vscode and filter per file in the extension. If there is a strong desire to use it with any file I can easily implement that behavior (as said, at the cost of having the extension awake while it might not be needed)
